src/InvoicePrinter.php

```diff
@@ -49,6 +49,7 @@ class InvoicePrinter extends FPDF
     public $footernote;
     public $dimensions;
     public $display_tofrom = true;
+    public $display_tofromheaders = true;
```

Comment:
please use camel case variable names for new elements. The previous one is kept as-is for compatibility only (and it's quite screwed up :smiley:  )

I suggest:

```php
protected $displayToFromHeaders = true;
```

Reply:
DONE

`protected $displayToFromHeaders = true;`